### PR TITLE
created tooltip

### PR DESCRIPTION
**Pull request by thinknoack**

### CODE REVIEW

File: packages/openneuro-components/src/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  flow: 'up' | 'down' | 'left' | 'right'
+}
+
+export const Tooltip = ({
+  children,
+  tooltip,
+  flow,
+  ...props
+}: TooltipProps) => {
+  const myRef = React.createRef<HTMLSpanElement>()
```
Comment (tranttommy) [inline packages/openneuro-components/src/Tooltip/Tooltip.tsx]: You should use the `useRef` hook instead for this

File: packages/openneuro-components/src/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+export interface TooltipProps {
+  tooltip: string
+  flow: 'up' | 'down' | 'left' | 'right'
+}
+
+export const Tooltip = ({
+  children,
+  tooltip,
+  flow,
+  ...props
```
Comment (tranttommy) [inline packages/openneuro-components/src/Tooltip/Tooltip.tsx]: What might the `...props` be used for?

File: packages/openneuro-components/src/Tooltip/Tooltip.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react'
+import './tooltip.scss'
+
+export interface TooltipProps {
+  tooltip: string
+  flow: 'up' | 'down' | 'left' | 'right'
+}
+
+export const Tooltip = ({
+  children,
```
Comment (tranttommy) [inline packages/openneuro-components/src/Tooltip/Tooltip.tsx]: Need to include children in the `interface`. The type for `children` would be `React.ReactNode`

File: packages/openneuro-components/src/Tooltip/Tooltip.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+const Template: Story<TooltipProps> = args => <Tooltip {...args} />
+
+export const Up = Template.bind({})
+Up.args = {
+  children: <>Lorem ipsum</>,
+  tooltip: 'this is the tooltip using the data attr',
+  flow: 'up',
+}
+Up.parameters = {
```
Comment (tranttommy) [inline packages/openneuro-components/src/Tooltip/Tooltip.stories.tsx:23]: What does this `parameters` property do?

**Review by tranttommy — CHANGES_REQUESTED** [review]

**Review by tranttommy — APPROVED** [review]
Tooltips look good, but we may want to merge in the Radio buttons branch before this. I would suggest not branching off another component branch next time to make things easier to review and merge.

**Review by thinknoack — COMMENTED** [review]
> Tooltips look good, but we may want to merge in the Radio buttons branch before this. I would suggest not branching off another component branch next time to make things easier to review and merge.

yep, thats how my other branches are. There was a reason I branched off of radio but I am not remembering why at the moment.

**Pull request merged**